File: python/fizzbuzz.py
def main():
    for i in range(1, 101):
        result = ""
        if i % 3 == 0:
            result += "Fizz"
        if i % 5 == 0:
            result += "Buzz"
        if result == "":
            result += str(i)
        print(result)

if __name__ == "__main__":
    main()

File: c/fizzbuzz.c
#include <stdio.h>

int main(int argc, char** argv) {
    for (int i = 1; i <= 100; i++) {
        if (i % 15 == 0) {
            puts("FizzBuzz");
        } else if (i % 3 == 0) {
            puts("Fizz");
        } else if (i % 5 == 0) {
            puts("Buzz");
        } else {
            printf("%d\n", i);
        }
    }
    return 0;
}